docs/source/faq.rst

```diff
@@ -107,8 +107,8 @@ Why I don't see any logs in the console?
 ----------------------------------------
 
 In version R19, Gunicorn doesn't log by default in the console.
```

Comment:
I'd say drop this sentence, move the last sentence to the top, and change it to start with "Starting with Gunicorn 19.2, [...]".